vendor/engines/projects/config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   resources :facilities, only: [] do
-    resources :projects, controller: "projects/projects", except: [:destroy]
+    resources :projects, controller: "projects/projects", except: [:destroy] do
+      collection do
```

Comment:
BTW this was the key to everything: putting `inactive` inside a `collection` block made `load_and_authorize_resource` do the right thing.